src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements Draw {
```

Comment:
```suggestion
public class Circle extends Figure {
```
all interfaces should be implemented in parent class